homeassistant/components/samsungtv/encrypted/authenticator.py

```diff
@@ -0,0 +1,316 @@
+"""SamsungTV Encrypted."""
+# flake8: noqa
+# mypy: ignore-errors
+# pylint: disable=[invalid-name,missing-class-docstring,missing-function-docstring]
+from __future__ import annotations
+
+import hashlib
+import logging
+import re
+import struct
+from typing import Any
+
+from Crypto.Cipher import AES
```

Comment:
Home Assistant uses `cryptography` 

There should be an almost identical interface available without the need to load another crypto lib

Reply:
I have adjusted to use cryptography (>= 35.0.0)